src/lib/Serialize.hs

```diff
@@ -79,6 +91,18 @@ prettyVal val = case val of
             variant = Variant (NoExt types) theLabel repeatNum value
         _ -> error "SumAsProd with an unsupported type"
     _ -> return $ pretty con
+  Record (LabeledItems row) -> do
+    let separator = line' <> ","
+    let bindwith = " ="
+    let elems = concatMap (\(k, vs) -> map (k,) (toList vs)) (M.toAscList row)
+    let fmElem = \(label :: Label, v) -> do
+          vStr <- prettyVal v
+          return $ pretty label <> bindwith <+> vStr
```

Comment:
If you want to be super fancy than this can be a one liner I think: `fmap (pretty label <> bindwidth <+>) $ prettyVal v` 😛

Reply:
Thanks for the tip! I applied a similar one-liner.

`hlint` suggested `((pretty label <> bindwith) <+>) <$> prettyVal v` instead of using `fmap`.

Reply:
`<$> = fmap` 😄